CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ this project adheres to
 ### Added
 
 - Tracing with a filter set by `RUST_LOG` environment variable.
+- Support passing SSH passphrase via `SSH_PASSPHRASE` environment variable.
```

Comment:
This sentence sounds a bit off. The imperative mood is commonly used in commit titles and issue descriptions, but we should be careful when using it in other contexts.